sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -62,7 +62,7 @@ export class ManagedIdentityCredential implements TokenCredential {
       _options = options;
     } else {
       // options only constructor
-      _options = options;
+      _options = clientIdOrOptions;
```

Comment:
Is this one of those situations where `clientId` can be passed in the options bag _or_ constructor? Do we need to set `this.clientId` in this case?

Reply:
Well, not in this PR. At the moment we don’t have a notion to pass the client Id through the options bag. The options bag is just a `TokenCredentialOptions`, which does not have a `clientId` property. I am adding `clientId` to the options bag as part of the changes of this PR: https://github.com/Azure/azure-sdk-for-js/pull/20043/files#diff-beb4dd9be20c512344b474fff258dc24e4863f3b84cd900e0f6be9b761f2ea7dR101-R103 since we’ll be working with resourceIds in the options bag as well.

Reply:
Ah that's where I saw that. No problem.